newrelic-java-agent/distributed-tracing/kafka-examples/README.md

```diff
@@ -111,7 +111,7 @@ In this screenshot the `kafka-producer` and `kafka-consumer` services are connec
 
```

Comment:
Can you make sure that the distributed trace screenshot (`kafka-distributed-trace.jpg`) is updated as well since it now references classes whose names have changed (e.g. `KafkaProducerController`)?